rgd/geodata/permissions.py

```diff
@@ -108,7 +108,7 @@ def filter_perm(user, queryset, role):
     # Check setting for unassigned permissions
     if settings.RGD_GLOBAL_READ_ACCESS:
         unassigned = queryset.filter(**{user_path + '__isnull': True}).distinct()
-        return unassigned | filtered
```

Comment:
this was previously evaluating the querysets and returning all records. We want this to remain as a queryset and let the views handle pagination